environment/src/main/java/com/sequenceiq/environment/environment/flow/creation/handler/freeipa/FreeIpaCreationHandler.java

```diff
@@ -367,6 +370,19 @@ private EnvCreationEvent getNextStepObject(EnvironmentDto environmentDto) {
                 .build();
     }
 
+    private void setImage(EnvironmentDto environment, CreateFreeIpaRequest createFreeIpaRequest) {
+        if (environment.getFreeIpaCreation() != null) {
+            String imageCatalog = environment.getFreeIpaCreation().getImageCatalog();
+            String imageId = environment.getFreeIpaCreation().getImageId();
+            if (!Strings.isNullOrEmpty(imageCatalog) && !Strings.isNullOrEmpty(imageId)) {
```

Comment:
if null should we fill up the request with default values ?

Reply:
If is already handled in freeipa service.

Reply:
maybe a log would be nice then

Reply:
fixed